app/views/collectibles/_collectible_form.html.erb

```diff
@@ -11,7 +11,7 @@
   </dd>
 <% end %>
   
-<dt><%= form.label :collections_to_add, ts("Add To Collections") %> <%= link_to_help "add-collectible-to-collection" %></dt>
+<dt><%= form.label :collections_to_add, ts("Add to collections") %> <%= link_to_help "add-collectible-to-collection" %></dt>
```

Comment:
If you're feeling especially motivated, this can be a good opportunity to move those into i18n files. See [here for the Archive target i18n standards](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards) and [here for a fresh example of such conversion](https://github.com/otwcode/otwarchive/pull/4639/files).

Reply:
Thank you for the suggestion - it gave me the confidence to try tackle this. I've done my best to follow what has been done in the past but very open to feedback on the naming and organisation of the i18n values.